### PR TITLE
Slave

### PR DESCRIPTION
**Pull request by thedhruvn**
Added "title" field for all McUser - default is blank, also allows for Dr., Mr., Mrs., and Ms. through a dropdown.

Title viewable only in the profile view (see changes to see how I added the title with the user.get_full_name_with_maiden() function)

Added Export to excel functionality with a new row for Titles - tested it and it works.

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 class McUser(models.Model):
   """Fields from default User model:
+    title
```
Comment (joshcai) [inline core/models.py]: Not needed - these fields were for the fields we were going to use from the base User model. If you could get rid of 'first_name' and 'last_name' that would be great, I forgot to remove them lol.

Comment (thedhruvn) [reply core/models.py]: Ok - I'll remove those lines.

**Review by joshcai — APPROVED** [review]
Looks good - thanks!

**Comment by atvaccaro** [comment]
Changed it to merge into develop instead of master.

**Pull request merged**